src/pages/home/report/ReportActionItem.js

```diff
@@ -118,7 +118,7 @@ class ReportActionItem extends Component {
                                 measureContent={() => (
                                     <ReportActionContextMenu
                                         isVisible
-                                        reportID={-1}
+                                        reportID={this.props.reportID}
```

Comment:
@roryabraham, to get it to work, I had to change this. Any reason NOT to do so?

Reply:
No, this is safe 👍